pages/productAppExtension/[productId]/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { H4, Panel, Text } from "@bigcommerce/big-design";
+import { useRouter } from "next/router";
+import ErrorMessage from "@components/error";
+import Loading from "@components/loading";
+import { useProductInfo } from "@lib/hooks";
+
+const ProductAppExtension = () => {
+    const router = useRouter();
+    const productId = Number(router.query?.productId);
+    const list = [];
```

Comment:
this isn't needed since we've made `list` optional in the hook (i.e. `list?:`). Remember, adding a question mark denotes it as optional